pyQuARC/code/custom_validator.py

```diff
@@ -132,16 +143,18 @@ def doi_missing_reason_explanation(explanation, missing_reason, doi):
     @staticmethod
     @if_arg
     def boolean_check(field_value):
+        # Checks if the value is a boolean, basically one of 'true' or 'false'
```

Comment:
What about TRUE or FALSE? or any variants?

Reply:
that's why there's a `field_value.lower()` in the check